### PR TITLE
TRIDENT-18400 - Update entity group for inventory

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) updates this article:
https://help.sumologic.com/docs/cse/records-signals-entities-insights/create-an-entity-group/

Issue number: [TRIDENT-18400](https://sumologic.atlassian.net/browse/TRIDENT-18400)

I originally did the work with PR #2910, but I published too early! So I reverted it with PR #3030. I'll publish this PR once the feature is truly ready to release.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft.

**Comment by jpipkin1** [comment]
This PR is ready for review.

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/cse/records-signals-entities-insights/create-an-entity-group.md [hunk 1]
```diff
@@ -92,12 +92,15 @@ Follow these instructions to create an Entity Group that corresponds to a group
 1. **Inventory Type**. Select one of:
     * Computer
     * User
+1. **Inventory Key**. Select an attribute to use from the **Inventory Type** selected above. Select **groups** if you want to use an existing Entity Group attribute. 
 1. **Source**. Select an inventory source from the pull-down list.
-1. **Group**. Enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group.
+1. **Value**. Enter a value for the attribute selected in the **Inventory Key** field above. If **groups** was selected in the **Inventory Key** field, enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group. (If you select the **Dynamic Schema Tags** checkbox below, you must enter *****.)
```
Comment (AngelATX) [inline docs/cse/records-signals-entities-insights/create-an-entity-group.md:97]: Small nit-pick:
Reads: (If you select the **Dynamic Schema Tags** checkbox below, you must enter *****.)

When `Dynamic Schema Tags` is selected, `*` will automatically be applied as stated correctly below on line 99: `If you select this option, the **Value** field changes to *****`. 

Might want to update this line, as it implies the user might be able to change this value when they cannot, and `*` is only a valid value when `dynamic schema tags` is checked.

Comment (jpipkin1) [inline docs/cse/records-signals-entities-insights/create-an-entity-group.md:97]:
```suggestion
1. **Value**. Enter a value for the attribute selected in the **Inventory Key** field above. If **groups** was selected in the **Inventory Key** field, enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group. 
```

Comment (jpipkin1) [reply docs/cse/records-signals-entities-insights/create-an-entity-group.md]: @AngelATX, per our Slack conversation, I removed 
>(If you select the **Dynamic Schema Tags** checkbox below, you must enter *)

**Review by AngelATX — APPROVED** [review]
Looks great! Thanks John. 🚢

**Pull request merged**